src/mesido/esdl/esdl_heat_model.py

```diff
@@ -1717,13 +1822,17 @@ def convert_electricity_cable(self, asset: Asset) -> Tuple[Type[ElectricityCable
         min_voltage = asset.in_ports[0].carrier.voltage
         max_current = max_power / min_voltage
         self._set_electricity_current_nominal_and_max(asset, max_current / 2.0, max_current)
+        length = asset.attributes["length"] if asset.attributes["length"] != 0.0 else 10.0
```

Comment:
I would suggest adding logger.warning(...) for when the 10.0 is used so that one is made aware of this when it happens

Reply:
updated